components/Nav/Nav.tsx

```diff
@@ -20,27 +21,33 @@ const Nav = () => {
   const [isNavVisibleOnMobile, toggleNavVisibility] = useToggle(false);
   const [isSticky] = useStickyElement();
 
+  const handleClickAway = () => {
+    if (isNavVisibleOnMobile) toggleNavVisibility();
+  };
+
   return (
     <NavBar {...{ isSticky }}>
       <HiddenMainPageTitle />
       <Brand />
-      <Container>
-        <NavLinks areVisible={isNavVisibleOnMobile}>
-          <NavMenu />
-          <SocialMedia />
-        </NavLinks>
-        <SeparatorHiddenOnMobile />
+      <ClickAwayListener onClickAway={handleClickAway}>
```

Comment:
IMHO, writing just `onClickAway={() => isNavVisibleOnMobile && toggleNavVisibility()}` would be faster to read than separate function. What do you think?

Reply:
It looks much cleaner :) i will replace it.